addon/components/osf-footer/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+import layout from './template';
+
+import config from 'ember-get-config';
+
+export default Ember.Component.extend({
+    layout,
+    helpUrl: config.OSF.helpUrl,
+    supportEmail: Ember.computed(function() {
+        return '<n uers=\"znvygb:pbagnpg@bfs.vb\" ery=\"absbyybj\">Pbagnpg</n>'.replace(
+            /[a-zA-Z]/g,
+            function(e) {
+                return String.fromCharCode((e <= 'Z' ? 90 : 122) >= (e = e.charCodeAt(0) + 13) ? e : e - 26);
```

Comment:
Given that the entire page is *already* constructed from JS, I'm not sure what ROT13 gets us here in terms of protecting email addresses.

However, it's not a blocker for the PR; I know we're hewing close to the original.